desktop/apps/consign/client/actions.js

```diff
@@ -144,10 +159,12 @@ export function completeSubmission () {
                           .send({ state: 'submitted' })
 
       dispatch(updateSubmission(submissionResponse.body))
+      dispatch(submissionCompleted())
       dispatch(stopLoading())
       dispatch(push(stepsConfig.thankYou.submissionPath.replace(':id', submissionResponse.body.id)))
     } catch (err) {
       dispatch(stopLoading())
+      dispatch(submissionError('convection_complete_submission'))
```

Comment:
That's just a way to identify where the error occurred (in this case, when the submission is _completed_, not created), so when we track it we can put a custom message. For analytics purposes the entire error string is not very helpful here (and kind of odd to format), so we pass this and in `analytics_middleware` handles them separately.